dvc/output/gs.py

```diff
@@ -1,8 +1,34 @@
 from __future__ import unicode_literals
 
+from dvc.path import GSPathInfo
 from dvc.remote.gs import RemoteGS
 from dvc.output.s3 import OutputS3
 
 
 class OutputGS(OutputS3):
     REMOTE = RemoteGS
+
```

Comment:
For the record: discussed introducing `PathInfo(scheme, ...)` function to help use common logic where possible.